docs/source/changelog.rst

```diff
@@ -9,6 +9,7 @@ Release date: TBD
 
 New features
 ------------
+- Support for julia `Project.toml`, `JuliaProject.toml` and `Manifest.toml` files.
```

Comment:
```suggestion
- Support for julia `Project.toml`, `JuliaProject.toml` and `Manifest.toml` files in :pr:`595` by
:user:`davidanthoff`
```

So we know who to blame ... I mean credit ;)

Reply:
Haha, sure :)